packages/datetime/src/dateRangePicker.tsx

```diff
@@ -56,6 +57,13 @@ export interface IDateRangePickerProps extends IDatePickerBaseProps, IProps {
     onChange?: (selectedDates: DateRange) => void;
 
     /**
+     * Called when the user changes the hovered date range, either from mouseenter or mouseleave.
+     * When triggered from mouseenter, it will pass the date range that would result from next click.
+     * When triggered from mouseleave, it will pass `[null, null]`.
```

Comment:
does it make sense that when im moving hovering between days, i keep getting `[null, null]` fired in between? genuine questions, not rhetoric